index.html

```diff
@@ -13,7 +13,7 @@
                 <img class="logo" src="logo1.png">
             <div class="forminput">
                 <label class="l"><img src="username.png"></label>
-                <input type="text" placeholder="Name">
+                <input type="text" placeholder="Fixoc">
```

Comment:
Looks like you first created this in master branch and then submitted change here. How am I supposed to review it like this? I can't see entire html code.